.github/workflows/run_unit_tests.yml

```diff
@@ -44,7 +44,7 @@ jobs:
         run: dotnet restore
 
       - name: Build & Run Unit Tests
-        run: dotnet test --verbosity minimal --no-restore --settings tests\UnitTests.runsettings --results-directory ${{ env.test_results_path }}
+        run: dotnet test --verbosity normal --no-restore --settings tests\UnitTests.runsettings --results-directory ${{ env.test_results_path }}
```

Comment:
Do we still want this at `normal`? It looks like a difference of 20K lines at normal vs 1K at minimal

Reply:
I can probably revert it - needed it to see where things were failing.